js/runner.go

```diff
@@ -427,6 +416,7 @@ func (u *VU) RunOnce(ctx context.Context) error {
 	}
 
 	// Call the default function.
+	ctx := context.Background()
```

Comment:
This doesn't seem right... the execution of the default function should happen with the `maxDurationCtx` of the executor, so it (especially any in-flight HTTP requests, WS connectons, etc.) aborts whenever that context is cancelled.

Reply:
Fixed